src/main/java/com/commercetools/sync/inventories/InventorySyncOptionsBuilder.java

```diff
@@ -33,14 +33,14 @@ public static InventorySyncOptionsBuilder of(@Nonnull final SphereClient ctpClie
     }
 
     /**
-     * Set option that indicates whether sync process should create supply channel of given key when it doesn't exists
+     * Set option that indicates whether sync process should create a supply channel of given key when it doesn't exist
      * in a target project yet. If set to {@code true} sync process would try to create new supply channel of given key,
-     * otherwise sync process would log error and fail to process draft with given supply channel key.
+     * otherwise sync process would log an error and fail to process draft with given supply channel key.
```

Comment:
otherwise sync process -> otherwise the sync process

Reply:
and fail to process draft with given supply channel key. -> and fail to process **the** draft with **the** given supply channel key.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/ac7e561a12b70d8d3c43ae1f8daf95f608542d38